config/config.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"log"
+)
+
+type Config struct {
+	Host             string `yaml:"host"`
+	Database         string `yaml:"database"`
+	User             string `yaml:"user"`
+	Sslmode          string `yaml:"sslmode"`
+	BinaryParameters string `yaml:"binary_parameters"`
+	Port             string `yaml:"port"`
+}
+
+func ParseConfig(path string) (*Config, error) {
+	bytes, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+	c := new(Config)
+	err = yaml.Unmarshal(bytes, c)
+	if err != nil {
+		return nil, err
+	}
+	return c, err
+}
+
+func Conninfo(path string) (conninfo string) {
```

Comment:
Can we make this a method on `*Config`? 

Then we can pass this config struct around, which makes adding config bits much easier.

Reply:
done.